scipy/stats/_stats_py.py

```diff
@@ -9718,6 +9718,472 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
     return SignificanceResult(statistic, pval)
 
 
+@dataclass
+class QuantileTestResult:
+    r"""
+    Result of `scipy.stats.quantile_test`.
+
+    Attributes
+    ----------
+    statistic: float
+        The statistic used to calculate the p-value; one of, `T1`, the observed
+        number of observations less than or equal to the hypothesized quantile,
+        or `T2`, the observed number of observations strictly less than the
+        hypothesized quantile. Two test statistics are required to handle the
+        possibility the data was generated from a discrete or mixed
+        distribution. `T1` is sensitive whether the population quantile is
+        significantly greater than the hypothesized value and `T2` to whether
+        the population quantile is signficantly less.
+
+    statistic_sign : int
+        One of `+1` or `-1` depending on which of `T1` or `T2` was used to
+        calculate the p-value respectively. `1` corresponds to the "greater"
+        alternative hypothesis and `-1` to the "less".  For the "two-sided"
+        alternative, a value of `+1` means, that given the data, there is
+        greater than equal likelihood that the population quantile is larger
+        than the hypothesized value, while a value of `-1` means there is
+        strictly greater likelihood that it is less than the hypothesized
+        value.
```

Comment:
I'm not familiar with this use of "likelihood". Also, what is the likelihood greater than?

Reply:
Sorry. It’s nonsense. I wrote it on the plane earlier today while dozing. I’ll fix it up.